src/main/java/thePackmaster/cards/cthulhupack/PageOfTheDead.java

```diff
@@ -14,17 +13,20 @@ public class PageOfTheDead extends AbstractCthulhuCard {
 
     public PageOfTheDead() {
         super(ID, 0, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
+        baseMagicNumber = magicNumber = 5;
+        cardsToPreview = new Lunacy();
 
-        cardsToPreview = new Necronomicurse();
     }
 
     public void use(AbstractPlayer p, AbstractMonster m) {
-        if (!upgraded) addToBot(new MakeTempCardInHandAction(new Necronomicurse()));
-        Wiz.applyToSelf(new PageOfTheDeadPower(p, 1));
-
+        loseSanity(magicNumber);
+        AbstractCard strike = Wiz.returnTrulyRandomPrediCardInCombat(card -> card.type == CardType.ATTACK
```

Comment:
It's possible to have a set of packs with no 2-cost attacks. Eldritch Horror itself doesn't have any, and it isn't too hard to find 6 other packs without 2-cost attacks. In fact, from a quick look through things, there's a _lot_ of packs without 2-cost attacks, so this is pretty likely to happen if you play enough. I ran into a crash for something similar to this yesterday -- I had a set of packs with only 1 rare attack, and I did something that assumed there would be at least 2 rare attacks.

If there are no 2-cost attacks in your packs, this will return null and crash. We should have some sort of handling for that -- I'm not sure what option is best. Creating nothing is simplest but feels pretty bad if you take this card without realizing you don't have any 2-cost attacks. An alternative is to fall back to creating a different card (like Hand of Greed, which I think is the only colorless 2-cost attack) or fall back to looking across all base game 2-cost attacks.

A corollary to this is that there are sets of packs that only have one 2-cost attack, where playing this will give you the same card every time. Don't know if you need to do anything about that, but figured I'd mention it.

Also I'd suggest renaming this to `attack` instead of `strike`.

Reply:
Falling back to making a Rummage is another option.

Reply:
This function does not go through the pack list, it pulls from all game cards.  Just tested to verify, got Brainwave while not using the Psychic pack.